docs/velero-backup.md

```diff
@@ -0,0 +1,11 @@
+# Steps to backup/restore Kubernetes resources with Velero
+
+### 1. Enable velero application
+
+TODO: Add information about how to configure Velero, and what needs to be in place for it to install properly.
+
+### 2. Run `kubectl get pods -n velero` & `velero version` to confirm successful installation on cluster and that velero CLI is installed locally. To install Velero CLI follow these instructions: https://velero.io/docs/v1.8/basic-install/#install-the-cli.
```

Comment:
Since backup is automatically done on schedule then restore procedure would be beneficial